hacking/pr_labeler/label.py

```diff
@@ -26,6 +26,14 @@
 LABELS_BY_CODEOWNER: dict[OwnerTuple, list[str]] = {
     ("TEAM", "@ansible/steering-committee"): ["sc_approval"],
 }
+RELEASE_MANAGEMENT_MEMBERS = {
+    "anweshadas",
+    "felixfontein",
+    "gotmax23",
+    "mariolenz",
+    "rooftopcellist",
+    "webknjaz",
+}
 HERE = Path(__file__).resolve().parent
```

Comment:
should we have some kind of team like the steering committee has so we don't have to edit this list directly if release managers come and go?

Reply:
We could create an @ansible/release-management-wg (release-management-wg is to disambiguate @ansible/ansible-release-managers) group, but group members aren´t public, so we wouldn´t be able to use that data here.

Reply:
> group members aren´t public, so we wouldn´t be able to use that data here.

Once we set up the authentication methods proposed in https://github.com/ansible/ansible-documentation/issues/382, we'll be able to access this data.